src/keymap_manager.cc

```diff
@@ -8,6 +8,10 @@
 namespace Kakoune
 {
 
+// the define-user-mode command will push into this vector
+Vector<String> keymap_modes = { "normal", "insert", "prompt", "menu",
```

Comment:
I really dont like this global, it needs to stay in sync with the 'KeymapMode` enum.